### PR TITLE
Fat 2717 c353645

### PR DESCRIPTION
**Pull request by TetianaParanich**
***
https://issues.folio.org/browse/FAT-2717
***
https://foliotest.testrail.io/index.php?/cases/view/353645
***

### CODE REVIEW

File: cypress/e2e/data-import/permissions/permission-only-viewing-settings.cy.js [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+  const matchProfileName = `C353645_match_profile_${getRandomPostfix()}`;
+  const jobProfileName = `C353645_job_profile_${getRandomPostfix()}`;
+
+  before('create test data', () => {
+    let mapProileId;
+    let actProfileId;
+    let matchProfileId;
+    cy.getAdminToken();
+    NewFieldMappingProfile.createMappingProfileViaApi(mappingProfileName)
+      .then((res) => {
```
Comment (IhorBohdan) [inline cypress/e2e/data-import/permissions/permission-only-viewing-settings.cy.js:38]: better named response as mappingProfileResponse,then res.

File: cypress/e2e/data-import/permissions/permission-only-viewing-settings.cy.js [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+        mapProileId = res.body.id;
+
+        NewActionProfile.createActionProfileViaApi(actionProfileName, mapProileId)
+          .then((resp) => {
```
Comment (IhorBohdan) [inline cypress/e2e/data-import/permissions/permission-only-viewing-settings.cy.js:42]: better named response as actionProfileResponse,then resp

File: cypress/e2e/data-import/permissions/permission-only-viewing-settings.cy.js [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+            actProfileId = resp.body.id;
+
+            NewMatchProfile.createMatchProfileViaApi(matchProfileName)
+              .then((response) => {
```
Comment (IhorBohdan) [inline cypress/e2e/data-import/permissions/permission-only-viewing-settings.cy.js:46]: the same

**Review by IhorBohdan — APPROVED** [review]

**Review by MaksimTerekhov — APPROVED** [review]

File: cypress/e2e/data-import/permissions/permission-only-viewing-settings.cy.js (hunk 1 above)
Comment (TetianaParanich) [reply cypress/e2e/data-import/permissions/permission-only-viewing-settings.cy.js]: Done

File: cypress/e2e/data-import/permissions/permission-only-viewing-settings.cy.js (hunk 2 above)
Comment (TetianaParanich) [reply cypress/e2e/data-import/permissions/permission-only-viewing-settings.cy.js]: Done

File: cypress/e2e/data-import/permissions/permission-only-viewing-settings.cy.js (hunk 3 above)
Comment (TetianaParanich) [reply cypress/e2e/data-import/permissions/permission-only-viewing-settings.cy.js]: Done

**Pull request merged**